app/models/hardware.rb

```diff
@@ -143,7 +143,7 @@ def m_vm(parent, xmlNode, _deletes)
       self.cpu_speed = e.attributes["cpuspeed"] if e.name == "cpuspeed"
       self.cpu_type = e.attributes["cputype"] if e.name == "cputype"
       parent.autostart = e.attributes["autostart"] if e.name == "autostart"
-      self.numvcpus = e.attributes["numvcpus"] if e.name == "numvcpus"
+      self.cpu_sockets = e.attributes["cpu_sockets"] if e.name == "cpu_sockets"
```

Comment:
@bdunne `e.attributes` and `e.name` should remain `numvcpu`.  That value comes from parsing the VMware config file directly or parsing another supported VM type here [VmConfig](https://github.com/ManageIQ/manageiq/tree/master/gems/pending/metadata/VmConfig).  This is similar to the change to `memory_mb` above from PR #5055 here [hardware.rb](https://github.com/ManageIQ/manageiq/pull/5055/files#diff-6d975a235a345bb9f0d3375e70aaf691R125).